examples/twittermap/web/public/javascripts/map/countmap.js

```diff
@@ -62,11 +62,19 @@ angular.module('cloudberry.map')
           ctrlClass.remove();
         }
       }
+      
+      function hideMapControl(name){
+        var ctrlClass = $("."+name);
+        if (ctrlClass) {
+          ctrlClass.hide();
+        }
+      }
 
       // remove CountMap controls
       removeMapControl('legend');
       removeMapControl('normalize');
       removeMapControl('sentiment');
+      hideMapControl('info');
```

Comment:
What's the difference between removeMapControl and hideMapControl? Why can't we use the currently existing method? :-)

Reply:
hideMapControl simply makes the div invisible by setting the hide option. It is easier to make the div reappear again later. The original removeMapControl method deletes the div entirely and create them later. Because the other 3 map controls are added in a different place with 'info', the code cannot be used in the first place, so I figured I can use a different approach to hide the div.

Reply:
I tried create/remove instead of show/hide for 'info' but there are some problems. Maybe we can discuss it face to face whenever you are available?

Reply:
@waans11 , could you approve this PR and merge it?